### DISCUSSION TITLE
Demo environment errors

### DISCUSSION THREAD

**Discussion by jurjendevries** [saleor/saleor#9123] [General]
When browsing to https://demo.saleor.io/ clicking a product for example https://demo.saleor.io/product/space-dust-navy-paint/62/ I am getting an error "Network error: Response not successful: Received status code 400" also the link to the dashboard https://demo.saleor.io/dashboard/ isn't responding.

Please let me know when it is up again, because I like to learn if product reviews are available or not. Thank you.

**Comment by karolkielecki** [comment]
Hi @jurjendevries, 
Saleor doesn't support product reviews by default. You'll have to create an integration with some other product review tool/platform. The link above is broken indeed, but when I was browsing our demo I didn't find any broken links there. When I clicked on the same product, everything was ok: https://demo.saleor.io/product/space-dust-navy-paint/

**Comment by jurjendevries** [comment]
Hi @karolkielecki thank you.
Here still an error.
I will attach a screenshot. Hopefully that helps someone fixing it.

![Screenshot 2022-02-15 13 18 53](https://user-images.githubusercontent.com/355488/153997585-8776df34-9d28-4b15-b6bb-8b23d00f319f.png)

**Reply by karolkielecki** [reply]
@jurjendevries Maybe it's an issue with the service worker. We won't fix it since we don't maintain saleor-storefront anymore. We're focusing on developing a new project react-storefront that is going to replace the old one. https://github.com/saleor/react-storefront
